classes/abstracts/ActionScheduler_Abstract_ListTable.php

```diff
@@ -104,10 +104,12 @@ abstract class ActionScheduler_Abstract_ListTable extends WP_List_Table {
 
 	/**
 	 * Makes translation easier, it basically just wraps
-	 * `_x` with some default (the package name)
+	 * `_x` with some default (the package name).
+	 * 
+	 * @deprecated 3.0.0
 	 */
 	protected function translate( $text, $context = '' ) {
```

Comment:
This method doesn't make any sense, it's using the `package` as text domain, but not possible to change, also if changed it, will also change some hooks in this class.